### PR TITLE
Create temple.json

### PR DESCRIPTION
**Pull request by temple69**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by sital002 — CHANGES_REQUESTED** [review]
Your filename must match your github username instead of ```data/temple.json``` it should be ```data/temple69.json```  
Please make the necessary changes to get your profile merged.

**Comment by temple69** [comment]
@sital002 I have made the necessary  changes

File: data/temple.json
Comment (kumarsonsoff3) [inline data/temple.json:1]: Please remove `temple.json`, as you've created 2 files with same content.

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on BioDrop, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at t[hese Docs](https://www.biodrop.io/docs/quickstart-json) which will help with the suggested changes the maintainers have proposed. 

Please remember you can also create your Profile using [Forms](https://www.biodrop.io/docs/quickstart-forms).

**Pull request closed without merging**